thrift/lib/rust/deterministic_hash/Cargo.toml

```diff
@@ -12,5 +12,5 @@ publish = false
 [dependencies]
 anyhow = "1.0.95"
 fbthrift = { version = "0.0.1+unstable", path = ".." }
-ring = "0.16.20"
+ring = "0.17.12"
```

Comment:
Let's go with `0.17.14` here instead, since we might as well get the perf improvements if we're already doing the upgrade.